zetaclient/config/types.go

```diff
@@ -88,7 +88,7 @@ type Config struct {
 
 	// chain configs
 	EVMChainConfigs map[int64]EVMConfig `json:"EVMChainConfigs"`
-	BitcoinConfig   BTCConfig           `json:"BitcoinConfig"`
+	BTCChainConfigs map[int64]BTCConfig `json:"BTCChainConfigs"`
```

Comment:
Ideally we would preserve the old field and merge it into the new map to make the upgrade a bit less painful

Reply:
The field `BitcoinConfig` is retained by commit https://github.com/zeta-chain/node/pull/2870/commits/b1ee2eb48b4db1debb851ba77fdf68819fd2feb7 to be backward compatible